### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by Feliverse**
### Portfolio: desktop version
**Learning objectives**

- Understand how to create UIs adaptable to different screen sizes using media queries.
- Understand how to parse a Figma design to create a UI.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,134 +14,150 @@
...
-        </article>
-        </div>
     </section>
 
 <!-- about section -->
 
     <section class="about">
+        <div class="about-cont">
 
+      <div class="about-head">
```
Comment (rskinnerc) [inline index.html:151]: ♻️ Awesome! You are listing the tech stack correctly 🎉. However, please note the skills section doesn't resemble the Figma Template on desktop devices (< 768px)

Currently, your design looks as follows:
image

- [ ] Please make sure you follow the original template layout 🙏. Seems like it turns into de desired layout once it hits 1015px


Figma template:
image

**Review by rskinnerc — CHANGES_REQUESTED** [review]
Hi @Feliverse 👋👋👋, 

Good job so far! 💫💫
There are some issues that you still need to work on to go to the next project but you are almost there!

giphy


## Highlights
- Great job laying out the elements of the page using grid and flexbox
- Awesome! Your linters are well-configured and running great

## Required Changes ♻️
_Check the comments under the review._
♻️ In general, your design looks good, however, seems like at the desired viewport size (768px) the content gets pulled to the left while the width of the full viewport is greater. Seems like you should check how the width of the elements and the margin is playing.

This is a 768px wide screen. Please note at the bottom how the scroll bar appears, if I scroll to the right, there is plenty of space.

Kindly, double-check how the width and the margins are being set so it becomes truly responsive at that mi size.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- Adding the correct anchor destination to the nav links. For example, you could add an `id` of `contact` to the contact section, and then in the nav link the href should be `#contact`, that way, whenever a user clicks any link, it would take the user to the desired section of the page.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Feliverse) [reply index.html:151]: Ups... I will adjunst

File: index.html [hunk 2]
```diff
@@ -14,134 +14,150 @@
             <a href="#"><img src="images/ic_menu.svg" alt="Menu"></a> 
         </nav>
     </header>
+
+     <!-- descktop version -->   
+
+     <header class="desk">
```
[the remark this answers is not in the document]
Comment (Feliverse) [reply index.html:20]: Thank you for it, I didn't see before

File: index.html [hunk 3]
```diff
@@ -168,7 +184,10 @@ <h4>Skills</h4>
                 <li class="inBox">CitLAb</li>
                 <li class="inBox">Terminal</li>
             </ul>
+          <div class="Box image-about"></div>
         </div> 
+      </div>
+    </section> 
 
         <!-- form section  -->
```
[the remark this answers is not in the document]
Comment (Feliverse) [reply index.html:192]: 🤳 I will find a solution, thank you

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @Feliverse , 

I am Favour 👩🏽 , your reviewer for this Pull Request.

Great Job Implementing all the required changes by the previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**